config/locales/views/admin/fr.yml

```diff
@@ -287,6 +287,17 @@ fr:
               podcast_admin: Podcast Admin
               restricted_liquid_tag: Restricted Liquid Tag
               single_resource_admin: Single Resource Admin
+              resource_admin_article: Article Admin
+              resource_admin_badge: Badge Admin
+              resource_admin_badge_achievement: Badge Achievement Admin
+              resource_admin_broadcast: Broadcast Admin
+              resource_admin_comment: Comment Admin
+              resource_admin_config: Config Admin
+              resource_admin_display_ad: Display Ad Admin
+              resource_admin_data_update_script: Data Update Script Admin
+              resource_admin_feedback_message: Feedback Message Admin
+              resource_admin_html_variant: HTML variant Admin
```

Comment:
nitpicky, but can we capitalize Variant here?

Reply:
its fixed @dukegreene 🤝